frontend/src/routes/index.js

```diff
@@ -5,7 +5,7 @@ import { useKeycloak } from '@react-keycloak/web';
 import HomePage from '../pages';
 import FormPage from '../pages/Form';
 import AdminPage from '../pages/admin';
-import ApplicationSummaryPage from '../pages/ApplicationSummary';
+import DetailsPage from '../pages/Details';
```

Comment:
do you prefer detailsPage over using applicationSummaryPage or was this a merge thing?